src/style.css

```
@@ -1,3 +1,80 @@
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  height: 40px;
+  width: 40px;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: block;
+  position: relative;
+  margin: 0 10px;
+}
+
+.nav__link {
+  display: block;
+  justify-content: center;
+  align-items: center;
+  padding: 23px 0;
+
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
```

Don;t forget to do variables for all colors
```suggestion
  color: ...;
```